packages/web/src/components/MenuDrawer.tsx

```diff
@@ -81,6 +87,15 @@ const MenuDrawer = ({ handleMenuClose, open }: MenuDrawerProps) => {
               <ListItemIcon>{PolicyIcon}</ListItemIcon>
               <ListItemText>Privacy policy</ListItemText>
             </ListItem>
+            <ListItem
+              button
+              component={RouterLink}
+              onClick={() => handleMenuClose()}
+              to="/favorite-resources"
```

Comment:
Sorry, missed in the issue but maybe just `/favorites`?

Reply:
5c5736fd15f220878bce6a238bc317e2305dee06